docs/content/tools/xdebug.md

```diff
@@ -95,6 +95,13 @@ Note: If `PHP_IDE_CONFIG=serverName=${VIRTUAL_HOST}` is set before web request d
 
 You can run your scripts in console and debug them in the same way as browser requests. For example, you can run drush command: `fin drush fl` and debug this drush command from the Features module.
 
+
+### Drush 9
+
+In order to use Drush 9 with XDebug you will need to run commands directly through the `exec` command to the bin directory where Drush is located.
+This can usually be done by running `fin exec -T vendor/bin/drush` from where the vendor directory is located instead of running the regular command 
```

Comment:
Why the -T ?

Reply:
This should have been removed some reason I thought the existing Drush command had the -T command.